README.md

```diff
@@ -15,69 +15,33 @@ The instructions below will assume you've successsfully cloned the project and n
 1. Add the Android platform:
 
    `$ cordova platform add android`
+
 1. Prepare:
 
    `$ cordova prepare`
 
 1. Build and run:
 
    ```
-   $ cordova build android
    $ cordova run android
    ```
 
 ## iOS Installation
+
 1. Add the iOS platform: 
 
    `$ cordova platform add ios`
 
-   Note: you should expect to see the following error in the command's output:
-
-   ```
-   Discovered plugin "cordova-plugin-foresee" in config.xml. Adding it to the project
-   Installing "cordova-plugin-foresee" for ios
-   Failed to install 'cordova-plugin-foresee': Error: pod: Command failed with exit code 1
-        at ChildProcess.whenDone (/Users/wayne.burkett/foresee/test/foresee-sdk-cordova-sample/SampleApp/platforms/ios/cordova/node_modules/cordova-common/src/superspawn.js:169:23)
-        at emitTwo (events.js:106:13)
-        at ChildProcess.emit (events.js:191:7)
-        at maybeClose (internal/child_process.js:877:16)
-        at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
-   Failed to restore plugin "cordova-plugin-foresee" from config.xml. You might need to try adding it again. Error: Error: pod: Command failed with exit code 1
-   ```
-
-   This is a [known Cordova issue](https://issues.apache.org/jira/browse/CB-13597), which we'll address in the next few steps.
 1. Prepare:
 
    `$ cordova prepare`
-1. Go to the iOS dir: 
-
-   `$ cd platforms/ios`
-1. Edit the  `Podfile` and modify the `platform` line to read `platform :ios, '10.0'`. (This addresses the issue noted above.)
-1. Update pods: 
 
-   `$ pod install`
-1. Go back to original folder: 
-
-   `$ cd ../..`
 1. Build and run:
 
    ```
-   $ cordova build ios
    $ cordova run ios
    ```
 
-   Or run:
-   
-   `$ cordova emulate ios`
-
-   As of Xcode 10.x you'll need to add the `UseModernBuildSystem=0` build flag: 
-
-   `$ cordova emulate ios --buildFlag="-UseModernBuildSystem=0"`
-
-   You'll probably also want to specify a target (e.g. `--target="iPhone-X, 12.0"`). Putting it all together:
-
-   `$ cordova emulate ios --buildFlag="-UseModernBuildSystem=0" --target="iPhone-X, 12.0"`
```

Comment:
So we just don't use the emulate command, and don't need the buildFlag? If so, we should align between the 2 apps